src/Modules/Mess/components/MessActivities.jsx

```diff
@@ -16,12 +17,13 @@ function MessActivities() {
     { title: "View Bill" },
   ];
 
+  // Function to handle tab changes
   const handleTabChange = (direction) => {
     const newIndex =
       direction === "next"
-        ? Math.min(+activeTab + 1, tabItems.length - 1)
-        : Math.max(+activeTab - 1, 0);
-    setActiveTab(String(newIndex));
+        ? Math.min(activeTab + 1, tabItems.length - 1)
+        : Math.max(activeTab - 1, 0);
+    setActiveTab(newIndex);
```

Comment:
Discard this

Reply:
if i remove this the slide button used to change tabs is not working